packages/nullable-array/nullable-array.0.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+name: "ocplib-nullable-array"
```

Comment:
This `name` field needs to be removed, as it doesn't match the directory name. The `version` field is also unnecessary.